File: src/main/java/toker/panel/repository/PanelUserRepository.java
package toker.panel.repository;

import toker.panel.entity.PanelUser;

public interface PanelUserRepository extends BaseRepository<PanelUser, Integer> {
    PanelUser findByUsername(String username);
    PanelUser findByClaimedIdentity(String claimedIdentity);
}

File: src/main/java/toker/panel/repository/PanelUserRepository.kt
package toker.panel.repository

import toker.panel.entity.PanelUser

interface PanelUserRepository : BaseRepository<PanelUser, Int> {
    fun findByUsername(username: String): PanelUser
    fun findByClaimedIdentity(claimedIdentity: String): PanelUser
}